test/quantization/core/test_workflow_module.py

```diff
@@ -1118,6 +1123,52 @@ def test_fused_mod_reduce_range(self):
         self.assertEqual(obs.quant_min, 0)
         self.assertEqual(obs.quant_max, 127)
 
+    def test_embedding_bag_qat_config(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.emb1 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                self.emb2 = torch.nn.EmbeddingBag(num_embeddings=10, embedding_dim=12,
+                                                  include_last_offset=True, scale_grad_by_freq=False, mode='sum')
+                # Note for EmbeddingBag, indices should never be quantized, only outputs
+                self.dequant = torch.quantization.DeQuantStub()
+
+            def forward(self, indices):
+                x = torch.cat((self.emb1(indices), self.emb2(indices)))
+                return self.dequant(x)
```

Comment:
Why do we need a dequant here? the outputs of the embedding lookup ops should be in float